middleware.ts

```diff
@@ -88,7 +88,18 @@ export default async function middleware(req: NextRequest) {
     (locale) => !pathname.startsWith(`/${locale}/`) && pathname !== `/${locale}`
   );
 
-  console.log('isLocaleMissing', isLocaleMissing);
```

Comment:
Please add the console log back. This is in order to detect problems in the middleware.